src/Tooltip.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <div>
```

Comment:
Same as `Popover.vue`: Change this to `<span>` so that the components stays on the same line.